Signal/src/ViewControllers/AdvancedSettingsTableViewController.m

```diff
@@ -204,25 +204,18 @@ - (void)updateTableContents
                                                          actionBlock:nil]];
 
     if (OWSSignalService.sharedInstance.isCensorshipCircumventionManuallyActivated) {
-        [censorshipSection addItem:[OWSTableItem itemWithCustomCellBlock:^{
```

Comment:
I've used this PR to consolidate around the `OWSTableItem` factory methods which DRY up a ton of code.